helm-charts/charts/vote-chart/templates/vote-deployment.yaml

```
@@ -0,0 +1,22 @@
+      app: {{ .Values.vote.name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.vote.name }}
+    spec:
+      containers:
+        - image: {{ .Values.vote.image }}
+          name: {{ .Values.vote.containerName }}
+          ports:
+            - containerPort: {{ .Values.vote.containerPort }}
+              name: vote
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)